docs/adr/0003-in-onderzoek-meeleveren-bij-gevraagde-velden.md

```diff
@@ -0,0 +1,145 @@
+- in onderzoek gegevens worden niet geleverd als het onderzoek is beëindigd (datum einde onderzoek is gevuld).
+
+## JSON voorbeelden voor een gevraagd veld met bijbehorend inOnderzoek veld  
+
+### gevraagd veld is een gegeven dat in onderzoek kan staan
+
+geboorte.datum veld wordt gevraagd en de hele groep geboorte staat in onderzoek (aanduiding: 010300)
+
+```
+{
+    "geboorte": {
+        "datum": ...,
+        "inOnderzoek": {
+            "datum": true,
+            "datumIngangOnderzoek": { ... }
+        }
+    }
+}
+```
+
+geboorte.datum veld wordt gevraagd en de geboorteplaats staat in onderzoek (aanduiding: 030320)
+
+```
+{
+    "geboorte": {
+        "datum": ...
+    }
+}
+```
+
+### gevraagd veld is afgeleid uit één gegeven dat in onderzoek kan staan
+
+leeftijd wordt gevraagd en de hele categorie persoon staat in onderzoek (aanduiding: 010000)
+
+```
+{
+    "leeftijd": ...,
+    "inOnderzoek: {
+        "leeftijd": true,
+        "datumIngangOnderzoek": { ... }
+    }
+}
+```
+
+### gevraagd veld is afgeleid uit meerdere gegevens van verschillende categoriën en deze gegevens kunnen in onderzoek staan
+
+adressering.aanschrijfwijze wordt gevraagd en hele groep naam (010200) en hele groep naam partner (050200) staan in onderzoek:
+
+```
+{
+    "adressering": {
+        "aanschrijfwijze": ...,
+        "inOnderzoek": {
+            "aanschrijfwijze": true
+        }
+    }
+}
+```
+
+### gevraagd veld is een object die is afgeleid uit gegevens die in onderzoek kunnen staan en het in onderzoek staan van deze gegevens is relevant bij het gebruik van het object
+
+gezag wordt gevraagd en hele groep familierechtelijke betrekking (026200) van ouder1 staat in onderzoek
+
+```
+{
+    "gezag": [
+        {
+            "type": "GezamenlijkGezag",
+            "minderjarige": {
+                ...
+            },
+            "ouder": {
+                ...
+            },
+            "derde": {
+                ...
+            },
+            "inOnderzoek": true
+        }
+    ]
+}
+```
+
+bewoning wordt gevraagd en datum aanvang adreshouding van de verblijfplaats van een bewoner staat in onderzoek
+
+```
+{
+    "bewoningen": [
+        {
+            "adresseerbaarObjectIdentificatie": ...,
+            "periode": { ... },
+            "bewoners: [
+                {
+                    "burgerservicenummer": ...,
+                    "inOnderzoek": true
+                }
+            ],
+            "mogelijkeBewoners": []
+        }
+    ]
+}
+```
+
+### gevraagd veld is een object die is afgeleid uit gegevens die in onderzoek kunnen staan maar het in onderzoek staan van deze gegevens is niet relevant bij het gebruik van het object
+
+bewoning wordt gevraagd en de straatnaam van de verblijfplaats van een bewoner staat in onderzoek
```

Comment:
dit is denk ik niet een heel realistisch voorbeeld. Misschien is "geslachtsnaam" of "geboortedatum" beter?

Reply:
aangepast